tests/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+#
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2021, Intel Corporation
+#
+
+include(cmake/ctest_helpers.cmake)
+
+set(LIBS_BASIC
+	uasync
+	${CMAKE_THREAD_LIBS_INIT})
+
+set(SOURCES_DUMMY_TEST
+	dummy.c)
+
+add_custom_target(tests)
+
+add_flag(-Wall)
+
+add_cstyle(tests-all
+		${CMAKE_CURRENT_SOURCE_DIR}/*/*.[ch])
+add_check_whitespace(tests-all
```

Comment:
I suggest add parametrized functions like in vmemcache:
add_check_whitespace, add_cstyle